index.js

```diff
@@ -1,6 +1,6 @@
 function stringIncludes(haystack, needle) {
-  for (let i = 0; i < haystack.length; i++) {
-    if (haystack.slice(i, needle.length) === needle) {
+  for (let i = 0; i < haystack.length + 1; i++) {
```

Comment:
This should be causing an off by 1 error. Adding 1 here pushes you past the end of the list.